rexmex/metrics/coverage.py

```diff
@@ -1,31 +1,54 @@
-from collections import Counter
-from typing import List
+from typing import List, Tuple
 
+import numpy as np
 
-def item_coverage(relevant_items: List, recommendations: List[List]) -> float:
+
+def user_coverage(possible_users_items: List[List], recommendations: List[Tuple]) -> float:
```

Comment:
can you give a more specific type annotation for `possible_users_items`? Is this a `List[List[int]]`?

Same for the recommentations

Reply:
I switched it to be a tuple since it always has to be of length 2. Added explicit types now. Users can use integers or strings, depending what they're using in their system.